### ISSUE TITLE
Missing image for default drive UUID

### ISSUE THREAD

**Issue by leforthomas** [cloudsigma/docker-machine-driver-cloudsigma#13]
If I run 

sudo docker-machine create --driver cloudsigma testinstance

as per documentation, I get an error with

[{Message:Object with uuid 6fe24a6b-b5c5-40ba-8860-771044d2500d does not exist Point: Type:notexist}]

I assume this is the default drive when not specified and it must have been removed somehow?

### Expected Behavior

A new machine should be created

### Current Behavior

Error message

[{Message:Object with uuid 6fe24a6b-b5c5-40ba-8860-771044d2500d does not exist Point: Type:notexist}]

### Steps to Reproduce
1. Just run the code mentioned above
2.
3.
4.

**Comment by bogdan-ds** [comment]
Thanks for reporting this @leforthomas, you're right. This is the UUID of a default drive used to create a test instance. Since the default drives vary across CloudSigma locations, it's not reliable to use a single UUID. We'll look into improving that.

**Comment by leforthomas** [comment]
Thanks Bogdan. I am having a lot of issues starting with my own image. I am basically stuck on the SSH bit, ie it cannot connect to the server. Is there something specific I need to do on the image to make it work? some steps or a working image would really help!

**Comment by bogdan-ds** [comment]
Hi @leforthomas, 

I suggest you get the UUID of some of the other preinstalled drives in the library and use that instead, it should work. Maybe choose a Ubuntu preinstalled image.

**Comment by leforthomas** [comment]
Sorry, how do you get the UUID from one of the library drives? I am on the library tab and when I select an image I can't see any UUID. The only way I can get a UUID is bny attaching it to a drive but I tried this already and it doesn't work.

**Comment by bogdan-ds** [comment]
Hi @leforthomas, 

Unfortunately we don't have the chance to debug this further now, but we'll try to add it to our schedules and let you know of a way to use the driver.

**Comment by pavel-github** [comment]
👋 @leforthomas,
we released the new version [v2.0.0](https://github.com/cloudsigma/docker-machine-driver-cloudsigma/releases/tag/v2.0.0) with the option `cloudsigma-drive-name`. Default value of this option is `ubuntu`.
This option introduces a new behavior:
- driver will search for all library drives containing this name and use the latest one for cloning the drive
- if you want to use specific library drive UUID, you have to "nullify" `cloudsigma-drive-name` option because only one option can be defined (the driver will print the warning message in this case).

An example from your previous comment would look so (I omit all default values):
1) using ubuntu 19 server with UUID 15e3a382-be4b-438e-a385-3493c39642c9
```
$ docker-machine create \
    --driver=cloudsigma \
    --cloudsigma-drive-name="" \
    --cloudsigma-drive-uuid=15e3a382-be4b-438e-a385-3493c39642c9 \
    --cloudsigma-drive-size=30 \
    myimagename
```
2) using ubuntu latest available image
```
$ docker-machine create \
    --driver=cloudsigma \
    --cloudsigma-drive-size=30 \
    myimagename
```

Let me know if v2.0.0 solves your issues. Thanks!
